### PR TITLE
Reformats ./packaging/installer/install-required-packages.sh with: shfmt -w -i 2 -ci -sr

### LINKED ISSUE #6967: Kickstart.sh doesn't work for CentOS 8 & RHEL 8
##### Bug report summary
Running the `kickstart.sh` on CentOS 8 fails on 'Installation of required packages'.

```
/etc/os-release information:
NAME            : CentOS Linux
VERSION         : 8 (Core)
ID              : centos
ID_LIKE         : rhel fedora
VERSION_ID      : 8

We detected these:
Distribution    : centos
Version         : 8
Codename        : 8 (Core)
Package Manager : install_yum
Packages Tree   : centos
Detection Method: /etc/os-release
Default Python v: 2

This system is receiving updates from Red Hat Satellite or Spacewalk server.
Last metadata expiration check: 2:42:06 ago on Fri Sep 27 14:58:16 2019.
No match for argument: Judy-devel
No match for argument: MySQL-python
No match for argument: autoconf-archive
No match for argument: autogen
No match for argument: libmnl-devel
No match for argument: libuv-devel
No match for argument: python
There are following alternatives for "python": python2, python36
No match for argument: python-psycopg2



We are very sorry!

Installation of required packages failed.

What to do now:

1. Make sure your system is updated.
Most of the times, updating your system will resolve the issue.


2. If the error message is about a specific package, try removing
that package from the command and run it again.
Depending on the broken package, you may be able to continue.


3. Let us know. We may be able to help.
Open a github issue with the above log, at:


https://github.com/firehol/netdata/issues                                                                                             
```
##### OS / Environment
CentOS 8
##### Netdata version (ouput of `netdata -V`)
1.10.0 (I am limited to this version due to company policies. Version should be irrelevant to this issue) 
##### Component Name
Installation scripts
##### Steps To Reproduce
`bash kickstart.sh all --non-interactive`
##### Expected behavior
Installation should succeed.

### PR DESCRIPTION
**Pull request by prologic**
Partially Fixes #6967

##### Summary

Code cleanup with agreed upon tooling `shfmt`

##### Component Name

area/packaging

##### Additional Information

1/3 PR(s) inbound

### CODE REVIEW

**Comment by Ferroin** [comment]
Looks good to me other than the complaints from shellcheck. Might make sense to just handle that all as part of this too.

**Comment by prologic** [comment]
> Looks good to me other than the complaints from shellcheck. Might make sense to just handle that all as part of this too.

It was handled in the next PR :)

File: packaging/installer/install-required-packages.sh [hunk 1]
```diff
@@ -129,1282 +128,1220 @@ EOF
...
+  ['alpine']="py-requests"
+  ['arch']="python2-requests"
+  ['centos']="python-requests"
+  ['debian']="python-requests"
+  ['gentoo']="dev-python/requests"
+  ['sabayon']="dev-python/requests"
+  ['rhel']="python-requests"
+  ['suse']="python-requests"
+  ['default']="python-requests"
+  [\
```
Comment (knatsakis) [inline packaging/installer/install-required-packages.sh]: This doesn't look right.

Comment (prologic) [reply packaging/installer/install-required-packages.sh]: Good catch! Uncertain how this happened; but fixed.

File: test.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```
Comment (knatsakis) [inline test.sh]: Did you maybe commit this file by mistake?

Comment (prologic) [reply test.sh]: Yes

**Pull request merged**